source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1149,7 +1246,7 @@ we distinguish two phases of connecting to a server and using it for application
 
 If there is a network error or timeout on the connection before the handshake completes,
 the client MUST replace the server's description
-with a default ServerDescription of type Unknown,
+with a default ServerDescription of type Unknown, with null topologyVersion,
```

Comment:
I think the "null topologyVersion" is implied by replacing the server description with a default `ServerDescription`?